x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/data/rule_migrations_data_lookups_client.ts

```diff
@@ -16,11 +16,12 @@ export class RuleMigrationsDataLookupsClient {
   constructor(
     protected currentUser: AuthenticatedUser,
     protected esScopedClient: IScopedClusterClient,
-    protected logger: Logger
+    protected logger: Logger,
+    protected spaceId: string
   ) {}
 
   async create(lookupName: string, data: LookupData): Promise<string> {
-    const indexName = `${LOOKUPS_INDEX_PREFIX}${lookupName}`;
+    const indexName = `${LOOKUPS_INDEX_PREFIX}-${this.spaceId}-${lookupName}`;
```

Comment:
I'm not sure if breaking compatibility with any existing usages is a concern or not, if it is, we can just do something as simple as 

```
const indexName = this.spaceId === default ? const indexName = `${LOOKUPS_INDEX_PREFIX}${lookupName}` : `${LOOKUPS_INDEX_PREFIX}${this.spaceId}-${lookupName}`;
```
just let me know what you think

Reply:
I would incline towards having `space name` in the index name, `default` or not. Helps users to clearly identify what data goes in where.

May be we write a migration from old indices to new one but i am not sure of the process 🤷 .

Reply:
will go with this as this const is not being used in any released version yet